includes/base/widget-base.php

```diff
@@ -774,4 +774,34 @@ public function remove_skin( $skin_id ) {
 	public function get_skins() {
 		return Plugin::$instance->skins_manager->get_skins( $this );
 	}
+
+	/**
+	 * @param string $plugin_title  Elementor-Pro | Elementor-Core
```

Comment:
Plugin title (not  "Elementor-Pro | Elementor-Core")

Reply:
done